### ISSUE TITLE
frobomind_make fails at line 143 during cut

### ISSUE THREAD

**Issue by madsherlock** [FroboLab/frobomind#14]
Fresh clone of frobomind and frobit_demo on Netrunner 14.2 (Ubuntu 14.04 derivative).
`$ cd <catkin_workspace>/src`
`$ frobomind/frobomind_make -i /frobit_demo`
Produces
`frobomind/frobomind_make: command substitution: line 143: syntax error near unexpected expression ")"`
`frobomind/frobomind_make: command substitution: line 143: 'echo $APP | cut -c1-1)'`
`No proper application defined: /frobit_demo
`

cut is version 8.21

I don't think the ")" is supposed to be in the cut expression on line 143 of frobomind_make.

However, this script did work for me before reinstallation of linux.

As far as I understand, the line command `echo $APP | cut -c1-1)` is supposed to produce a "/" for frobomind applications. However, it only works for me if I remove the parenthesis `echo $APP | cut -c1-1`.

**Comment by KjeldJensen** [comment]
You are right, the ")" is a bug, though it appears to work just fine in many situations. Can you please send a pull request?

FYI I have a new version of the frobomind_make under test, which supports auto completion etc. But it won't be available until late January.
